.github/workflows/update_h5p.yml

```diff
@@ -0,0 +1,104 @@
+name: Update H5P JS library
+
+on:
+  schedule:
+    # Runs at 00:00 UTC on Wednesday
+    - cron: '0 0 * * 3'
+  # Optional: Allow manual triggering
+  workflow_dispatch:
+
+jobs:
+  check-commits:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: 'develop'
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Get latest commit from target repo
+        id: get-commit
+        uses: octokit/request-action@v2.x
+        with:
+          route: GET /repos/{owner}/{repo}/branches/{branch}
+          owner: h5p
+          repo: h5p-php-library
+          branch: master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check commit status
+        id: check-commit
+        run: |
+            import os
+            from pathlib import Path
+            # Get the latest commit from the API response
+            latest_commit = "${{ fromJson(steps.get-commit.outputs.data).commit.sha }}".strip()
+            # Check stored commit
+            commit_file = Path('packages/hashi/.h5p-commit-sha')
+            stored_commit = ""
+            if commit_file.exists():
+                stored_commit = commit_file.read_text().strip()
+                has_changed = stored_commit != latest_commit
+            else:
+                print("No stored commit found, treating as new")
+                has_changed = True
+            # Set outputs for GitHub Actions
+            with open(os.environ['GITHUB_OUTPUT'], 'a') as f:
```

Comment:
I love all of this injected Python stuff, but this in particular is really neat to write to the env var as a file to speak to the GH action runner.